syles.css

```diff
@@ -0,0 +1,54 @@
+#title {
+    text-align: center;
+}
+
+#survey-form {
+    text-align: center;
+    width: 40%;
+    background-color: grey;
+    margin: auto;
+    padding: 20px;
+    border-color: aqua;
+    border-radius: 10px;
+
```

Comment:
Please add two spaces of indentation. Do not use the tab key, that is a CSS best practice 👍